.github/workflows/ci.yaml

```diff
@@ -5,7 +5,38 @@ on:
     branches:
       - "main"
 jobs:
-  test:
+  check:
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.18"
+      - name: Lint
+        run: make check
+      - name: Check that 'func.yaml schema' is up-to-date
+        run: make schema-check
+      - name: Check embedded templates content
+        run: go test -run "^\QTestFileSystems\E$/^\Qembedded\E$"
+
+  test-unit:
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.18"
+      - uses: actions/setup-java@v1
```

Comment:
Why are we setting up Java in CI when we aren't running template tests?

Reply:
Where did you get this code from?

Reply:
Maybe it was an unintentional checkin?
https://github.com/knative/func/commit/cc049952bb025a93e942279615563d4ee75352b2
Perhaps it was there during testing to run Quarkus template tests?

Reply:
Yeah it was already in the ci.yaml body. I just moved it around. I will remove from this PR.